.pre-commit-config.yaml

```
@@ -14,3 +14,7 @@ repos:
     -   id: check-yaml
         args: [--allow-multiple-documents]
     -   id: end-of-file-fixer
+-   repo: https://github.com/segmentio/golines
+    rev: v0.12.2
+    hooks:
+      - id: golines
```

How about at least setting the max-length to 120 chars?
```suggestion
        - id: golines
          args: [--max-len=120]      
```

---

done with [2d49aae](https://github.com/utilitywarehouse/tflint-ruleset-kafka-config/pull/10/commits/2d49aae1bb2d4f67a66ab212fd1b938383768ed6)